playbooks/dcn/deploy-dcn.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Deploy DCN environment
+  hosts: localhost
+  vars:
+    _arch_repo_path: /home/zuul/src/github.com/openstack-k8s-operators/architecture
+    _cifmw_repo_path: /home/zuul/src/github.com/openstack-k8s-operators/ci-framework
+    _jinja_template_src: "{{ _cifmw_repo_path }}/roles/ci_gen_kustomize_values/templates/dcn"
+    _arch_deployment_path: "{{ _arch_repo_path }}/examples/dt/dcn"
+  tasks:
+    - name: Load reproducer-variables
+      ansible.builtin.include_vars:
+        file: "~/reproducer-variables.yml"
+
+    - name: Load networking-environment-definition
+      ansible.builtin.include_vars:
+        file: "/etc/ci/env/networking-environment-definition.yml"
+        name: cifmw_networking_env_definition
+
+    - name: Get OpenShift access token
+      kubernetes.core.k8s_auth:
+        host: "{{ cifmw_openshift_api }}"
+        username: "{{ cifmw_openshift_user }}"
+        password: "{{ cifmw_openshift_password }}"
+        validate_certs: no
+      register: _auth_results
+
+    - name: Create a network subnet list
+      ansible.builtin.set_fact:
+        _network_ranges: >-
+          {{
+            cifmw_networking_env_definition.networks
+            | dict2items
+            | selectattr('key', 'search', '^ctlplane')
+            | map(attribute='value.network_v4')
+            | list
+          }}
+
+    - name: Deploy EDPM
+      include_tasks: deploy-edpm.yml
```

Comment:
Can this `include_tasks` be changed into an `include_role` which includes a new role like `ci_dcn_site`.

Then move all the j2 files in this PR and the `deploy-edpm.yml` tasks file into the new `ci_dcn_site` role?